telemetry/roles/prometheus_gaudi/tasks/deploy_gaudi_metric_exporter.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+
+# Assuming Prometheus was deployed and name space monitoring was created
+
+- name: Define base URL variable
+  set_fact:
+    base_url: "https://vault.habana.ai/artifactory/gaudi-metric-exporter/yaml/"
```

Comment:
all this manifest files should be part of telemetry.json  and use downloaded manifest file path for the deploying. 
 example: habana_device_plugin_yaml_url: "{{ hostvars['localhost']['offline_manifest_path'] }}/habana-device-plugin.yaml"

Reply:
done

Reply:
After moving the manifest files to telemetry.son, they would be downloaded (to offline_manifest_path) only if telemetry was added to input/software_config.json manually. We may need to let end users know this prerequisite for the role prometheus_gaudi (by adding documentation?).

Reply:
user has keep telemetry in software_config.json and run local repo to mirror all the necessary pks and images. post that user will invoke telemetry.yaml. this is the same flow for all the software stack in omnia.